packages/ui/src/common/components/forms/Calendar/Calendar.tsx

```diff
@@ -0,0 +1,97 @@
+import { addDays, startOfMonth, startOfWeek } from 'date-fns'
+import React, { useMemo } from 'react'
+
+import { Arrow } from '@/common/components/icons'
+import { DateSelection, PartialDateRange } from '@/common/types/Dates'
+import { earliest, fromRange, latest } from '@/common/utils/dates'
+
+import {
+  CalendarMonth,
+  CalendarWeekDay,
+  CalendarHeaderContainer,
+  CalendarDay,
+  CalendarTitle,
+  CalendarNav,
+} from './components'
+
+const DAYS = 6 * 7
```

Comment:
I'm not sure what `DAYS` represents. Could you rename it to something more meaningful?